docs/docs/get_started/configuration.md

```diff
@@ -398,6 +398,15 @@ Commit-Boost supports hot-reloading the configuration file. This means that you
 docker compose -f cb.docker-compose.yml exec cb_signer curl -X POST http://localhost:20000/reload
 ```
 
+### Signer module reload
+
+The signer module takes 2 optional parameters in the JSON body:
+
+- `jwt_secrets`: a string with a comma-separated list of `<MODULE_ID>=<JWT_SECRET>` for all modules.
+- `admin_secret`: a string with the secret for the signer admin JWT.
+
+In the case that someone of those isn't present, that parameter won't be updated.
```

Comment:
We discussed this out-of-band a bit; the current implementation of `reload` is going to refresh the JWT secrets and Admin secret from disk if they aren't provided in the call, so it will throw away anything done with the `revoke` route. If that's not intended then `revoke` would need some way to write persistent data to disk which isn't currently possible.

Anyway assuming it *is* intended, then let's just clarify here by replacing this line with the following:

"Parameters that are not provided will not be updated; they will be regenerated using their original on-disk data as though the signer service was being restarted. Note that any changes you made with calls to `/revoke_jwt` or `/reload` will be reverted, so make sure you provide any modifications again as part of this call."